akka-remote/src/main/scala/akka/remote/artery/compress/TopHeavyHitters.scala

```diff
@@ -38,10 +38,9 @@ private[remote] final class TopHeavyHitters[T >: Null](val max: Int)(implicit cl
   /** Returns copy(!) of items which are currently considered to be heavy hitters. */
   def snapshot: Array[T] = {
     val snap = Array.ofDim(max).asInstanceOf[Array[T]]
-    var i: Int = 0
-    val heapIterator = heap.reverseIterator
-    while (heapIterator.hasNext) {
-      val index = heapIterator.next()
+    var i = 0
+    while (i < max) {
+      val index = heap(i)
```

Comment:
When I updated the tests I expected it to be sorted on highest value, but that was not the case throughout all the updates.

So, as we'll anyways go through the entire heap, and the order doesn't really matter I'll revert to the 0 - max loop.